plugins/org.jkiss.dbeaver.ui.editors.data/src/org/jkiss/dbeaver/ui/data/managers/ContentValueManager.java

```diff
@@ -82,7 +82,7 @@ public void run() {
                 });
             }
             // Logo can be changed
-            manager.add(new Action("Open in external editor", DBeaverIcons.getImageDescriptor(UIIcon.DOTS_BUTTON)) {
+            manager.add(new Action("Open in external editor", DBeaverIcons.getImageDescriptor(UIIcon.FILE)) {
```

Comment:
"Open in external editor" can be localized :smiling_imp:

Reply:
Also what about "// Logo can be changed" comment then? You changed the logo.

Reply:
Good point